### PR TITLE
Added nf-test for metaphlan/makedb

### PR DESCRIPTION
**Pull request by GallVp**
Added nf-test for metaphlan/makedb with #6226

### CODE REVIEW

**Comment by LilyAnderssonLee** [comment]
Can you remove the [pytest](https://github.com/nf-core/modules/tree/master/tests/modules/nf-core/metaphlan/makedb) for this module?

File: modules/nf-core/metaphlan/makedb/main.nf [hunk 1]
```diff
@@ -6,9 +6,11 @@ process METAPHLAN_MAKEDB {
         'https://depot.galaxyproject.org/singularity/metaphlan:4.0.6--pyhca03a8a_0' :
         'biocontainers/metaphlan:4.0.6--pyhca03a8a_0' }"
```
Comment (LilyAnderssonLee) [inline modules/nf-core/metaphlan/makedb/main.nf:7]: Can you update the version to the latest: 4.1.1?

Comment (GallVp) [reply modules/nf-core/metaphlan/makedb/main.nf]: Thank you @LilyAnderssonLee 

That's done now!

**Review by LilyAnderssonLee — APPROVED** [review]
Well done! It looks good to me.

**Pull request merged**